### ISSUE TITLE
Wagtail 4.0 compatibility

### ISSUE THREAD

**Issue by djch** [neon-jungle/wagtail-metadata#79]
I'm sure this is planned anyway, but seeing as nobody's made an issue yet... this is the error I get running Wagtail 4.0rc1 with wagtail-metadata at the moment:

```shell
File "/blah/site-packages/wagtailmetadata/models.py", line 4, in <module>
    from wagtail.wagtailadmin.edit_handlers import FieldPanel, MultiFieldPanel
ModuleNotFoundError: No module named 'wagtail.wagtailadmin'
```

Looks like the new module path is [`wagtail.admin.edit_handlers`](https://github.com/wagtail/wagtail/blob/main/wagtail/admin/edit_handlers.py).

**Comment by djch** [comment]
Never mind. Looks like the code's been updated for this but is unreleased?

**Comment by seb-b** [comment]
@djch what version are you using? `4.0.0` should have the update module paths included

**Comment by djch** [comment]
Yeah, I'm on 4.0.0 now. Was a bit confused as the latest release isn't tagged on GitHub.com

Here's what I get trying to update to Wagtail 4.0rc2 with wagtail-metadata:
```shell
CRITICAL:pipenv.patched.pip._internal.resolution.resolvelib.factory:Cannot install -r /var/folders/zg/xrlkpcps18n88qvxpv87fmmh0000gn/T/pipenvw9v5g7kfrequirements/pipenv-qfswu94l-constraints.txt (line 4) and wagtail==4.0rc2 because these package versions have conflicting dependencies.
``` 

This doesn't happen if I remove wagtail-metadata as dependency.

**Comment by seb-b** [comment]
ah yep, there's a hard requirement on `3.x` for metadata, we should make it `>3.0` going forward since versioning in wagtail has changed to be closer to semver

**Comment by seb-b** [comment]
@djch I've just released `4.0.1`, let me know if that works for you
